core/src/alpenglow_consensus/vote_certificate.rs

```diff
@@ -141,10 +143,9 @@ impl VoteCertificate for CertificateMessage {
             self.bitmap.set(vote_message.rank as usize, true);
             // aggregate the signature
             // TODO(wen): put this into bls crate
```

Comment:
nit: we should remove the TODO above.

Reply:
Sorry I missed this, I'll leave a note to delete this on a next bls PR.